test/AudioTest.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import chai, {expect} from 'chai';
+import {shallow, mount} from 'enzyme';
+import Audio from '../src/components/Audio';
+import Transformation from '../src/components/Transformation';
+
+chai.use(require('chai-string'));
+
+describe('Audio', () => {
+  it("should include child transformation for a single source type", function () {
+    let tag = shallow(
+      <Audio cloudName='demo'
+             sourceTypes={'webm'}
+             publicId='dog'
+             sourceTransformation={{
+               webm: {overlay: 'text:verdana_30:webm!'}
```

Comment:
Kinda funny use-case... text overlay over an Audio tag. You can add a video layer